src/webroutes/fxserver/commands.js

```diff
@@ -68,10 +68,16 @@ module.exports = async function FXServerCommands(ctx) {
     //==============================================
     }else if(action == 'admin_broadcast'){
         if(!ensurePermission(ctx, 'players.message')) return false;
-        let cmd = formatCommand('txaBroadcast', ctx.session.auth.username, parameter);
-        ctx.utils.logCommand(cmd);
-        let toResp = await globals.fxRunner.srvCmdBuffer(cmd);
-        return sendAlertOutput(ctx, toResp);
+        const config = globals.configVault.getScoped('monitor')
```

Comment:
In txAdmin we would usually do `globals.fxRunner.config.xxxxxx` instead of calling the config vault.
i know this is not ideal, but that's how our scoped settings work.